internal/featuregates/mapper.go

```diff
@@ -0,0 +1,60 @@
+package featuregates
+
+import (
+	"bytes"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/api/features"
+)
+
+const (
+	// Add your new upstream gate here
+	ForceSemverUpgradeConstraintsGate = "ForceSemverUpgradeConstraints"
```

Comment:
I think I'd leave the actual code completely empty of upstream/downstream feature gates since:
- the only downstream feature gate we have right now is enabled in GA (so any upstream features it maps to would also be suddenly enabled in GA, which we don't want)
- we don't actually want to enable this particular feature gate at all (As you mentioned, I fully removed it upstream earlier this week)

What I would suggest is some comments in the various places with obvious dummy values like
```
// MyUpstreamFeature = "MyUpstreamFeature"
...
// features.FeatureGateNewOLMDownstreamFeature: {MyUpstreamFeature},
```